data-migration/2.x-to-3.x/README.md

```diff
@@ -0,0 +1,81 @@
+# Updating data from 2.x to 3.x
+
+The following code snippet emulates the situation where the code is in version 3.0.0 while the data is in version 2.0.2
+(this assumes that you've followed the [instructions for removing local
+state](https://github.com/research-software-directory/research-software-directory/blob/3.0.0/docs/dev.md#removing-local-state)):
+
+```shell
+# give me the code state for 3.0.0 ...
+git checkout 3.0.0
```

Comment:
Add fetch for safety so when `3.0.0`  tag exists remotely it can actually be found locally as well.
```suggestion
git fetch --tags
git checkout 3.0.0
```

Reply:
added